### DISCUSSION TITLE
blocks

### DISCUSSION THREAD

**Discussion by madhupashish** [community/community#58493] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

why i am blocked i am a contributer ?
please tell me 
github why u blocked me ?
i am a contributer

**Comment by queenofcorgis** [comment]
Hi there @madhupashish ,

Thanks for reaching out! If you were blocked from GitHub it is likely due to violating the [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) (spam, advertising, nonsensical/off topic post, etc.) GitHub placed you on a temporary restriction from GitHub Community. This restriction lasts seven days from when you received the email notification and will otherwise not affect your GitHub account. The GitHub Support and Community teams are unable to shorten this ban. Future violations of the [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) can result in an indefinite block, so be sure to familiarize yourself with the Code of Conduct.

**Comment by madhupashish** [comment]
i haven't done anything bad 
i was just contributing
